### ISSUE TITLE
Support `VK_EXT_layer_settings`

### ISSUE THREAD

**Issue by interacsion** [vulkano-rs/vulkano#2554]
`VK_EXT_validation_features` was [deprecated last year](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_validation_features.html#_deprecation_by_vk_ext_layer_settings), in favor of `VK_EXT_layer_settings`. Is there any plan to support it?

**Comment by mickvangelderen** [comment]
We can write

```rust
vulkano::instance::InstanceExtensions {
    ext_layer_settings: true,
    ..Default::default()
};
```

today, does that mean that this issue has been resolved?

**Comment by interacsion** [comment]
No, since afaik there's no way to actually specify the layer settings.
